app/gateway-oss/2.4.x/index.md

```diff
@@ -28,7 +28,7 @@ quickstart, the complete {{site.base_gateway}} getting started guide provides
 in-depth examples, explanations, and step-by-step instructions, and explores
 Kong's many available tools for managing the gateway.
 
-[quickstart]: /{{page.kong_version}}/getting-started/quickstart
-[configuring-a-service]: /{{page.kong_version}}/getting-started/configuring-a-service
-[enabling-plugins]: /{{page.kong_version}}/getting-started/enabling-plugins
+[quickstart]: /gateway-oss/{{page.kong_version}}/getting-started/quickstart
+[configuring-a-service]: /gateway-oss/{{page.kong_version}}/getting-started/configuring-a-service
+[enabling-plugins]: /gateway-oss/{{page.kong_version}}/getting-started/enabling-plugins
 [getting-started]: /getting-started-guide/latest/overview
```

Comment:
some links use `{{page.kong_version}}` and some have `latest`. Should we be consistent and try to avoid unneccessary redirects, but perhaps as part of different pr.

Reply:
Good point @bungle. That last one wasn't broken so I didn't fix it, and I think the overview doesn't exist anymore, hence the redirect. cc: @lena-larionova